web/src/components/software/SoftwarePage.tsx

```diff
@@ -111,6 +152,7 @@ function SoftwarePage(): React.ReactNode {
           <GridItem sm={12}>
             <IssuesHint issues={issues} />
           </GridItem>
+          <GridItem sm={12}>{reload}</GridItem>
```

Comment:
NP: I would render the GridItem only if needed. I.e., I'd move such a wrap to line 142 above. Or goes for something slightly more readable like 

```js
const showReposAlert = repos.some((r) => !r.loaded);

...

{ showReposAlert && <GridItem sm={12}><ReloadSection/><GridItem>}